HelpCentre/QA/flows/scheduled-topics-no-response-followup.md

```diff
@@ -0,0 +1,258 @@
+  aaq_metadata = parse_json("@contact.aaq_metadata")
+  inbound_id = aaq_metadata.inbound_id
+  feedback_secret_key = aaq_metadata.feedback_secret_key
+
+  # Fetch and store the error message, so that we don't need to do it for every error card
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_error_handling_button"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  # We get the page ID and construct the URL, instead of using the `detail_url` directly, because we need the URL parameter for `get` to start with `https://`, otherwise stacks gives us an error
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  button_error_text = page.body.body.text.value.message
+end
+
+card GotoMainMenu do
+  log("Going to Main Menu")
+  run_stack("7b50f9f4-b6cf-424b-8893-8fef6d0f489b")
+end
+
+card GotoFAQTopicsList do
+  log("Going to GotoFAQTopicsList")
+  update_contact(navbypass: "FAQTopicsList")
+  run_stack("7b50f9f4-b6cf-424b-8893-8fef6d0f489b")
+end
+
+card GotoHelpCentre do
+  log("Going to GotoHelpCentre")
+  update_contact(navbypass: "HelpCentre")
+  run_stack("7b50f9f4-b6cf-424b-8893-8fef6d0f489b")
+end
+
+```
+
+## Information not helpful
+
+```stack
+card TopicsNoResponseFollowup, then: TopicsNoResponseFollowupError do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "plat_help_topics_no_response_follow_up"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  callback_conf_msg = page.body.body.text.value.message
+  button_labels = map(page.body.body.text.value.buttons, & &1.value.title)
+
+  buttons(
+    TopicsNoResponseFollowupYes: "@button_labels[0]",
+    TopicsNoResponseFollowupNo: "@button_labels[1]"
+  ) do
+    text("@callback_conf_msg")
+  end
+end
+
+card TopicsNoResponseFollowupError, then: TopicsNoResponseFollowupError do
+  buttons(
+    TopicsNoResponseFollowupYes: "@button_labels[0]",
+    TopicsNoResponseFollowupNo: "@button_labels[1]"
+  ) do
+    text("@button_error_text")
+  end
+end
+
+```
+
+## Information was helfpul
+
+```stack
+card TopicsNoResponseFollowupYes, then: TopicsNoResponseFollowupYesError do
+  feedback_result =
+    put(
+      "https://hub.qa.momconnect.co.za/api/v1/inbound/feedback",
```

Comment:
Is there a reason why we're going through the hub for AAQ calls? Surely it would be better to directly integrate with the AAQ API, that way we don't need the hub (which is momconnect specific) for our platform implementations, only the AAQ service

Reply:
I'm happy to leave it going through the hub for now then. When there's a new AAQ API, we can then look at switching to using that directly instead.